### PR TITLE
Minor improvement for installation guide

### PR DESCRIPTION
**Pull request by ahaxu**
* Minor improvement for installation guide
* Always get the latest tag
* Update find path
* Copy binary files to ~/.local/bin

### CODE REVIEW

File: doc/getting-started/install.md [hunk 1]
```diff
@@ -159,9 +159,10 @@ Install the newly built node and CLI commands:
 
 If this doesn't work, you can manually copy the executable files to the `~/.local/bin` directory. Replace the place holder `<TAGGED VERSION>` with your targeted version:
 
-    cp -p dist-newstyle/build/x86_64-linux/ghc-8.10.2/cardano-node-<TAGGED VERSION>/x/cardano-node/build/cardano-node/cardano-node ~/.local/bin/
-
-    cp -p dist-newstyle/build/x86_64-linux/ghc-8.10.2/cardano-cli-<TAGGED VERSION>/x/cardano-cli/build/cardano-cli/cardano-cli ~/.local/bin/
+    # need to stop your current cardano node before cp-ing the bin file, and restart cardano node after then
+    export TAGGED_VERSION=$(git describe --tags)
+    cp $(find ~/cardano-node/dist-newstyle/build -type f -name "cardano-node" | grep "$TAGGED_VERSION") /usr/local/bin
```
Comment (rdlrt) [inline doc/getting-started/install.md:164]: Copying to /usr/local/bin may need root, does not necessarily work for everyone, and should not be recommendation from official doco

Comment (ahaxu) [reply doc/getting-started/install.md:164]: should we use `~/.local/bin/` instead of `/usr/local/bin` ?

Comment (ahaxu) [reply doc/getting-started/install.md:164]: >Copying to /usr/local/bin may need root, does not necessarily work for everyone, and should not be recommendation from official doco
but I don't know why. can you explain why it's not necessary for everyone and should not be recommend ? Or, you just feel like that ?

Comment (ahaxu) [reply doc/getting-started/install.md]: just updated destination dir to `~/.local/bin`

**Comment by newhoggy** [comment]
Thanks for the PR.  The changes are superseded by https://github.com/input-output-hk/cardano-node/pull/2624

**Pull request closed without merging**